src/Library/demos/Revealer/main.blp

```diff
@@ -0,0 +1,60 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: _("Revealer");
+  description: _("Animates the transition of its child from invisible to visible");
+
+  Box {
+    orientation: vertical;
+    spacing: 24;
+
+    ToggleButton button {
+      halign: center;
+      label: _("Show");
+
+      styles ["pill"]
+    }
+
+    Box {
+      spacing: 12;
+      orientation: vertical;
+
+      Adw.Clamp {
+        maximum-size: 400;
+
+        Revealer revealer {
+          transition-duration: 300;
+          transition-type: slide_up;
+
+          ListBox {
```

Comment:
When clicking on the button, it looks buggy because the cursor is above the first item which gets a special style for hover

can you either disable that or use an other child type? 

![Screenshot from 2023-06-17 21-27-36](https://github.com/sonnyp/Workbench/assets/19673/d91d6ed9-2508-4c85-85a5-fc20890932fd)